src/source/tile_cache.js

```diff
@@ -191,6 +191,8 @@ class TileCache {
     /**
      * Remove entries that do not pass a filter function. Used for removing
      * stale tiles from the cache.
+     *
+     * @param filterFn filter function to check.
```

Comment:
I'm not really sure how to describe this param.

Reply:
What about describing what the function signature would be? Worthwhile information in that case would be what it should return and in each case how it would impact the filtering.

It's taking as input a `Tile`, and the function `filterFn` controls whether the tile would be filtered _out_ during the filter process. What about:

```
@param filterFn filter function, takes as input a Tile, and controls whether the tile would be filtered out. If the function returns false, the tile would be filtered out, it won't otherwise.
```

Feel free to make it more succinct.